scout/commands/update/panel.py

```diff
@@ -46,10 +60,47 @@ def panel(panel, version, update_date, update_version):
             LOG.warning(err)
             click.Abort()
 
+    # Any mintainer updates?
+
+    new_maintainer = None
+    if add_maintainer:
+        user_obj = adapter.user(user_id=add_maintainer)
+        if not user_obj:
+            # Check if maintainers exist in the user database
+            LOG.warning(
+                "Maintainer user id {0} does not exist in user database".format(
+                    add_maintainer
+                )
+            )
+            click.Abort()
+
+        new_maintainer = panel_obj.get("maintainer", [])
+        if add_maintainer in new_maintainer:
+            LOG.warning("User {} already in maintainer list.".format(add_maintainer))
+            click.Abort()
+
+        new_maintainer.append(add_maintainer)
+
+    if revoke_maintainer:
+        current_maintainers = panel_obj.get("maintainer", [])
+        try:
+            current_maintainers.remove(revoke_maintainer)
+            new_maintainer = current_maintainers
+        except ValueError:
+            LOG.warning(
+                (
+                    "Maintainer user id {} is not a maintainer for panel {}.".format(
```

Comment:
I think you should always use `%s` for LOG messages. Learned that from pylint

Reply:
Ah, ok! I thought they did the same thing; cast the list into string and output, except that .format doesn't have the tuple issues that % has. Did you know what the advantage is? Hint: the formatting is may not have to happen! Thank you!